### PR TITLE
Update _index.es.md

### PR DESCRIPTION
**Pull request by AlexAndradeNet**
Translation to Spanish.

I've agreed the CLA

### CODE REVIEW

File: docs_source_files/content/grid/_index.es.md [hunk 1]
```diff
@@ -4,30 +4,14 @@ chapter: true
...
-The hub has a list of registered servers that it provides access to,
-and allows us to control these instances.
+Con Selenium Grid, un servidor actúa como el centro que enruta los comandos de prueba con formato JSON a uno o más nodos de Grid registrados. Las pruebas se ponen en contacto con el concentrador (o _hub_) para obtener acceso a instancias de explorador remoto. El concentrador tiene una lista de servidores registrados a los que proporciona acceso y nos permite controlar estas instancias.
 
-Selenium Grid allows us to run tests in parallel on multiple machines,
-and to manage different browser versions and browser configurations centrally
-(instead of in each individual test).
+Selenium Grid nos permite ejecutar pruebas en paralelo en múltiples máquinas, y administrar diferentes versiones y configuraciones de navegador de forma centralizada (en lugar de en cada prueba individual).
+
+Selenium Grid no es una bala de plata. Resuelve un subconjunto de problemas comunes de delegación y distribución, pero, por ejemplo, no administrará su infraestructura y podría no satisfacer sus necesidades específicas.
```
Comment (diemol) [inline docs_source_files/content/grid/_index.es.md]: `bala de plata` is really a Spanish expression? Could you please replace it with something more understandable?

**Review by diemol — CHANGES_REQUESTED** [review]

**Review by AlexAndradeNet — COMMENTED** [review]
The "bala de plata" (silver bullet) expression was changed for a more comprehensible frase

File: docs_source_files/content/grid/_index.es.md (hunk 1 above)
Comment (AlexAndradeNet) [reply docs_source_files/content/grid/_index.es.md]: The "bala de plata" (silver bullet) expression was changed for a more comprehensible frase

**Review by diemol — APPROVED** [review]
Thank you @AlexAndradeSan!

**Pull request merged**